src/anchor/anchor.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-anchor($status: primary, $underline: 'false') {
+  --_iui-anchor-popout-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z' /></svg>");
+
+  @include iui-anchor-status($status);
+  border-radius: $iui-border-radius;
+  text-underline-offset: 1px;
```

Comment:
Most image changes are a result of this line of CSS.  This makes the underline align with the bottom of the [descender](https://lh3.googleusercontent.com/XN6ATtQ7Q_NxH5Eb_ZMiAZI6Wh61nClqWKHyx10vhIly8lzEOKmswVEAxjQOzldyCJ_q5W6dTznM0fn5wS8ArPzhpDq4vfhUxnOz=w1064-v0).

Previously:
![Screen Shot 2022-03-28 at 4 34 56 PM](https://user-images.githubusercontent.com/849817/160482586-4abde5e2-9d7a-4410-8151-c9daa5ed4090.png)

Now:
![Screen Shot 2022-03-28 at 4 34 39 PM](https://user-images.githubusercontent.com/849817/160482602-81e0ce49-95dd-4cbe-9397-67b90723de6c.png)

Reply:
What if I remove just this one line & we review it.  Then once everything is good, I add that line back in.  That way we know any additional image changes are only the underline change.

Reply:
I've removed it for now but would like to add it back in at the end.